_posts/2018-06-17-message-calls-in-ethereum.md

```diff
@@ -58,49 +58,53 @@ EVM has 4 types of message call instructions:
 
 #### CALL
 
-`CALL` is the most straightforward type of message call. It takes takes 7 arguments, the last four of which are used for reading input data from the memory and writing result of the execution to the memory. The first three arguments:
+`CALL` is the most straightforward type of message call. It takes 7 arguments, the last four of which are used for reading input data from the memory and writing result of the execution to the memory. The first three arguments:
```

Comment:
add a the to this sentence - "data from the memory and writing **the** result of the execution to the memory